### ISSUE TITLE
Getting Double Notifications

### ISSUE THREAD

**Issue by kekeseb** [ricott/homey-com.volvocars#11]
I have a simple flow sending me notifications about the car charging cable
![Screen Shot 2021-02-26 at 14 27 02](https://user-images.githubusercontent.com/165168/109305934-c39bb880-783e-11eb-8814-1bcf40719f57.png)

I am however receiving every status change twice:
![Screen Shot 2021-02-26 at 14 25 50](https://user-images.githubusercontent.com/165168/109306047-e3cb7780-783e-11eb-9ce3-b9124ac3dd5e.png)
![Screen Shot 2021-02-26 at 14 26 00](https://user-images.githubusercontent.com/165168/109306061-e6c66800-783e-11eb-8c8e-930476f13a18.png)

**Comment by ricott** [comment]
Interesting, I just managed to replicate the problem.
Have not yet understood why since the trigger is only kicked off once as per log below

```
2021-03-03 17:18:07 [log] [ManagerDrivers] [voc] [0] [V60 / .... / ....] Updating capability 'charge_cable_status' from 'Connected, no Power' to 'Connected, with Power'
2021-03-03 17:18:07 [log] [ManagerDrivers] [voc] Triggering flow 'trigger.charge_cable_status_changed' with tokens { charge_cable_status: 'ConnectedWithPower' }
2021-03-03 17:18:07 [log] [ManagerDrivers] [voc] - device trigger for  V60 / ... / ....
```

**Comment by ricott** [comment]
There is now a test version that should address the issue, please test it and let me know.
https://homey.app/a/com.volvocars/test/

**Comment by ricott** [comment]
Havent heard anything so I assume this was solved in v1.1.4
